### DISCUSSION TITLE
Binding buttons to a controller?

### DISCUSSION THREAD

**Discussion by Kaje68** [Albeoris/Memoria#270] [Q&A]
Hi all, using memoria for FFVI PR with my primary need being the fast forward option.  The config files are easy enough for binding to a keyboard, but is there a way to bind it to a controller?

**Comment by Albeoris** [comment]
> Hi all, using memoria for FFVI PR with my primary need being the fast forward option. The config files are easy enough for binding to a keyboard, but is there a way to bind it to a controller?

I guess, you need to ask it [here](https://github.com/Albeoris/Memoria.FFPR).
But yes, you can bind controller by using "Actions" (like Menu+Cancel). But no you cannot bind controller buttons directly.
